stdlib/public/runtime/SwiftRT-ELF.cpp

```diff
@@ -34,14 +34,32 @@ DECLARE_SWIFT_SECTION(swift5_fieldmd)
 DECLARE_SWIFT_SECTION(swift5_assocty)
 DECLARE_SWIFT_SECTION(swift5_replace)
 DECLARE_SWIFT_SECTION(swift5_replac2)
+DECLARE_SWIFT_SECTION(swift5_builtin)
+DECLARE_SWIFT_SECTION(swift5_capture)
 }
 
 #undef DECLARE_SWIFT_SECTION
 
 namespace {
-static swift::MetadataSections sections{};
+static swift::MetadataSections sections{ swift::CurrentSectionMetadataVersion };
```

Comment:
Can you put an explanation for why you are initializing this?  This is unnecessary technically.

Reply:
Since we're asm-ing things here now anyway, maybe we can replace the dynamic initialization below with static data here too?

Reply:
1. @compnerd  yeah, i can add an explanation why I'm doing this static initialization. 
Basically the primary reason why I changed this - this one-line adjustment simplified life a bit for me (in particular, made debugging slightly easier). Static globals by default (i.e. without explicit initializers) are initialized with zeros and are placed into .bss (instead of .data). Since in the tests we already had some code which handled .data (while "emulating" dynamic loading) this change made it work automatically. 
2. @jckarter, probably we can, let me try / double check